lenra-io/devtool
# feat: Choose a user during the Devtool login process

## About this PR

Closes #211 


## Technical highlight/advice
The fake Hydra endpoint now manage views to rendre heex files.

The user id given through the socket overrides the token one to make it easy for Lenra client.

## How to test my changes
Run an app with the devtool following the README instructions and you'll see the user choosing page.

## Checklist

### I included unit tests that cover my changes

### I added/updated the documentation about my changes
